build.gradle

```diff
@@ -173,6 +174,38 @@ version = '0.0.1-SNAPSHOT'
 
 description = ''
 
+project.ext.diffChangelogFile = 'src/main/resources/config/liquibase/changelog/' + new Date().format('yyyyMMddHHmmss') + '_changelog.xml'
+if (!project.hasProperty('runList')) {
+    project.ext.runList = 'main'
+}
+
+liquibase {
+    activities {
+        main {
+            driver ''
+            url ''
+            username 'timeline'
+            password ''
+            changeLogFile 'src/main/resources/config/liquibase/master.xml'
+            defaultSchemaName ''
+            logLevel 'debug'
+            classpath 'src/main/resources/'
+        }
+        diffLog {
+            driver ''
+            url ''
+            username 'otp'
+            password ''
+            changeLogFile project.ext.diffChangelogFile
+            referenceUrl 'hibernate:spring:com.icthh.xm.ms.timeline.domain?dialect=&amp;hibernate.physical_naming_strategy=org.springframework.boot.orm.jpa.hibernate.SpringPhysicalNamingStrategy&amp;hibernate.implicit_naming_strategy=org.springframework.boot.orm.jpa.hibernate.SpringImplicitNamingStrategy'
```

Comment:
What does this line mean?

Reply:
autogenerated code by jhipter, copy pasted from otp ms
Used by liquibase gradle plugin to generate a changelog from diffs between a database and persistence entities in spring

username for database